pkg/server/definition_test.go

```diff
@@ -19,711 +19,714 @@ type definitionResult struct {
 	targetSelectionRange protocol.Range
 }
 
-func TestDefinition(t *testing.T) {
-	testCases := []struct {
-		name     string
-		filename string
-		position protocol.Position
+type definitionTestCase struct {
```

Comment:
Click on `Hide whitespace` when reviewing this file. There are no changes to the test cases